### ISSUE TITLE
Batch normalization layer from Keras only converts to BatchNormalization op if dims >= 4

### ISSUE THREAD

**Issue by ewenw** [onnx/tensorflow-onnx#2219]
Hi Onnx team, I'm running into a model conversion issue for Kera's BatchNorm layer. The BatchNorm layer is only converted into a BatchNorm Op in Onnx when input Dim>=4.  I believe this stems from the fact that the tf-onnx converter will only convert as a BatchNorm Op if the original TF Op is a FusedBatchNorm operator. Currently, my work around is to expand_dim the input so that the BatchNorm Op is inserted by the converter, otherwise it just becomes a single Mul op. This is problematic for training.

With 4D input:
<img width="206" alt="MicrosoftTeams-image (1)" src="https://github.com/onnx/tensorflow-onnx/assets/12105532/42d64ff2-b00e-49d7-9f5a-d0960c5a8ed7">

With 3D input:
<img width="219" alt="MicrosoftTeams-image (2)" src="https://github.com/onnx/tensorflow-onnx/assets/12105532/6c5e00e4-fee7-4ae8-addb-442e98573396">

**Comment by fatcat-z** [comment]
Is that possible to share a code snippet for a repro?
